addons/sourcemod/scripting/ttt/ttt_radio.sp

```diff
@@ -75,7 +75,7 @@ public Action Command_Radio(int client, int args)
 	}
 	else if (role == TTT_TEAM_TRAITOR)
```

Comment:
You should just move the role == TTT_TEAM_TRAITOR to the if before and remove the complete block.

Reply:
How is it now? I believe I fixed it.